### DISCUSSION TITLE
如何能让DatePicker在没有初始值的情况下，日期选择面板打开时显示的是某一个指定的时间？

### DISCUSSION THREAD

**Discussion by vivianus-chan** [ant-design/ant-design#33750] [General]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
比如DatePicker没有设置初始值的情况下，弹出面板会默认定位当前时间附近。而现在我限制了只能选择‘2020-10-01’到‘2020-10-31’之间的日期，其他时间全是禁止状态，当用户打开日期选择面板时，会看到全是禁止状态，而不知道可选范围在哪。

### What does the proposed API look like?
所以，是否可以在没有赋值的情况下，让面板弹出的时候就滚动到可以选择的时间范围，而不是当前时间附近。

**Comment by afc163** [comment]
Try `defaultPickerValue`

**Reply by vivianus-chan** [reply]
可以 谢谢
